README.md

```diff
@@ -370,21 +357,28 @@ The `none`, `left`, and `right` objects may also contain two additional properti
 
 ```json
 {
-    "gamepad" : {
-        "id" : "motion-controller-id",
-        "handedness" : {
-            "none" : {
-                "asset" : "some-url",
-                "root" : "none-handedness-controller",
-                "components" : [0],
-                "primaryButtonComponent" : 0,
-                "primaryAxisComponent" : 1
-            }
+    "handedness" : {
+        "none" : {
+            "asset" : "some-url",
+            "root" : "none-handedness-controller",
+            "components" : [0],
+            "primaryButtonComponent" : 0,
+            "primaryAxisComponent" : 1
         }
     }
 }
 ```
 
+## Top-level Properties
+The schema defines two key top-level properties: `version` and `id`.  The `version` property is a semantic version made up of a major, minor, and patch version number.  The major and minor version parts must always be up to date with the library being used to parse the mapping file.  The patch portion of the version may be out of sync.  The `id` property must match the `Gamepad.id` of the `Gamepad` object the mapping is being applied to.  The naming conventions for the `WebXR Device API` are concretely defined; the naming conventions for `WebVR` are somewhat more haphazard.
```

Comment:
> The schema defines two key top-level properties: `version` and `id`.

There's more top level properties than this, though, right? Especially since everything in the readme has been moved up a level in this PR.

Reply:
I also corrected the version language to match what we decided on yesterday